packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -164,9 +164,15 @@ const runMigrationCommand: CliCommandDefinition<CreateFlags> = {
 
     const projectConfig = apiClient({
       requireUser: true,
-      requireProject: true,
+      requireProject: false,
     }).config()
 
+    if (!project && projectConfig.projectId) {
```

Comment:
Shouldn't this be reversed? 
```suggestion
    if (!project && !projectConfig.projectId) {
```

Reply:
Yes, thank you.